hummingbot/core/data_type/order_book_tracker.py

```diff
@@ -136,11 +139,34 @@ def stop(self):
         if self._order_book_snapshot_router_task is not None:
             self._order_book_snapshot_router_task.cancel()
             self._order_book_snapshot_router_task = None
+        if self._update_last_trade_prices_task is not None:
+            self._update_last_trade_prices_task.cancel()
+            self._update_last_trade_prices_task = None
         if len(self._tracking_tasks) > 0:
             for _, task in self._tracking_tasks.items():
                 task.cancel()
             self._tracking_tasks.clear()
 
+    async def _update_last_trade_prices_loop(self):
+        '''
+        Updates last trade price for all order books, this is fall-back mechanism for when the web socket
+        has not yet started receiving messages or fails for some reasons.
+        '''
+        await wait_til(lambda: len(self.data_source._trading_pairs) == len(self._order_books.keys()))
+        while True:
+            try:
+                for trading_pair, order_book in self._order_books.items():
+                    if order_book.last_trade_price_updated >= time.perf_counter() - (60. * 3):
+                        continue
+                    last_price = await self.data_source.get_last_traded_prices([trading_pair])
```

Comment:
Also, this will result in unnecessary multiple call to the ticker endpoint for connectors like huobi and eterbase. I see those connectors supply last price of all available pairs in a single call. Since by design, `get_last_traded_prices()` takes in a list of pairs, why not just do away with the for-loop on line 158 above and simply pass all trading pairs at once?

Reply:
`last_trade_price_updated` is per order book, hence I used in the for-loop, and so far we don't have any strategy/configuration that uses multiple order book from the same exchange yet.
Though, your comment is valid, I will update this part to check for the oldest `last_trade_price_updated` and update them all.

Reply:
Nice work!